app/views/core/you_need_to_file.scala.html

```diff
@@ -51,7 +51,7 @@ <h1 class="govuk-heading-xl">@messages("ssttp.you-need-to-file.title")</h1>
         </div>
     </details>
     <section class="govuk-body">
-        <h3>@messages("ssttp.call-us.opening-times.heading")</h3>
+        <h2>@messages("ssttp.call-us.opening-times.heading")</h2>
```

Comment:
It looks like in all other eligibility kick out pages the 'Opening times' line is not a h2, but normal text. Should we change it to h2 or keep it consistent with the other pages?

Reply:
The other pages don't have H2 or H3 tags